core/src/test/java/org/infinispan/persistence/support/AsyncStoreTest.java

```diff
@@ -389,6 +389,7 @@ public void testEndToEndPutPutPassivation() throws Exception {
       doTestEndToEndPutPut(true);
    }
 
+   @Test(invocationCount = 100)
```

Comment:
Was this meant to be included as part of this commit?

Reply:
Nope :sheep: